Cargo.toml

```diff
@@ -79,7 +79,7 @@ jemallocator = { version = "0.3", features = ["profiling"] }
 jemalloc-ctl = "0.3"
 json-patch = "0.3"
 jsonschema = "0.30.0"
-jsonwebtoken = { version = "9", default-features = false }
+jsonwebtoken = { version = "9" }
```

Comment:
please specify the explicit new features that you need (do you use pem decoding?)